internal/gitinterface/hash.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package gitinterface
+
+import (
+	"encoding/hex"
+	"errors"
+)
+
+const (
+	zeroSHA1HashString   = "0000000000000000000000000000000000000000"
+	zeroSHA256HashString = "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Is this preperation for hash agility? If so, will this only be implmented with the git binary?

Reply:
yep!